### PR TITLE
Nfdiv 1520

### PR DESCRIPTION
**Pull request by velimd**
### Change description ###

Enter a description.
image

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-1520

**Before merging a pull request make sure that:**

- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

**Review by benouaer — APPROVED** [review]

**Comment by velimd** [comment]
closing due to the ticket hasnt translated the error messages and the ticket is blocked due to that.

File: src/main/steps/applicant1/confirm-your-joint-application/content.ts [hunk 1]
```diff
@@ -172,18 +171,178 @@ const en = ({ isDivorce, partner, userCase, isApplicant2, isJointApplication }:
...
+      ? `Mae Ceisydd 1 wedi rhoi manylion am achosion llys eraill sy’n ymwneud â’r ${
+          isDivorce ? 'briodas' : 'bartneriaeth sifil'
+        }:` +
+        '<br>' +
+        userCase.applicant1LegalProceedingsDetails +
+        '<br><br>' +
+        `Mae Ceisydd 2 wedi rhoi manylion am achosion llys eraill sy’n ymwneud â’r ${
+          isDivorce ? 'briodas' : 'bartneriaeth sifil'
+        }:` +
+        '<br>' +
```
Comment (benouaer) [inline src/main/steps/applicant1/confirm-your-joint-application/content.ts:239]: why do we add br tags in content, why not handle that in the template
I guess you're just replicating what's in the english version

Comment (velimd) [reply src/main/steps/applicant1/confirm-your-joint-application/content.ts:239]: yep

Comment (velimd) [reply src/main/steps/applicant1/confirm-your-joint-application/content.ts:239]: as mentioned before, i can clean up but i think theres alot to clean up so itll need its own ticket imo.

**Pull request merged**